Application/Application.iml

```diff
@@ -8,14 +8,5 @@
     <orderEntry type="inheritedJdk" />
     <orderEntry type="sourceFolder" forTests="false" />
     <orderEntry type="library" name="gson-2.8.2" level="project" />
-        <orderEntry type="module-library" exported="">
-      <library> 
-        <CLASSES> 
-          <root url="jar://$MODULE_DIR$/lib/gson-2.8.2.jar!/" /> 
```

Comment:
jesus christ

Reply:
just let it be removed... noone needs it anymore